CHANGELOG.md

```diff
@@ -0,0 +1,479 @@
+# Changelog
+
+## Version 2.5.8 (September 27, 2019)
```

Comment:
nit: I think it's a bit nicer to put the dates on a separate line and use ISO 8601 date format for everything for consistency ([workflow-cps](https://github.com/jenkinsci/workflow-support-plugin/blob/master/CHANGELOG.md), [workflow-support](https://github.com/jenkinsci/workflow-support-plugin/blob/master/CHANGELOG.md)):

```
## 2.5.8

Release date: 2019-09-27
```

Reply:
I can do that.

Of course, if you happen to know of a fancy automatic way to do that so it's not by hand, that'd be better :) These are like they were before on the wiki, and the converter just spit them back out as-is.

Reply:
Yeah, you might be able to do it with a regex magic, but generally I just do it by hand. In this case you have some dates using the full month, and others are abbreviated, so probably quicker in the end to just do it manually. Just a nit though, not a blocker or something that I think you _have_ to do.

Reply:
my goto regex to do exactly this in intellij idea:
`(###\s*\S*)\s*\((.*)\)
$1\n$2`